README.md

```diff
@@ -2,6 +2,31 @@
 
 ![Microverse](https://img.shields.io/badge/Microverse-blueviolet)
 
+The Budget app keeps track is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+Features: 
+- Register and log in, so that the data is private to them.
+- Introduce new transactions associated with a category.
+- See the money spent on each category.
+
+### Video introduction
+
+![video] (https://drive.google.com/file/d/1Ju8bL0QMtSG0qKpDTbWxa3Jkqyn0-tOR/view?usp=sharing)
```

Comment:
## Optional
- [ ] I suggest making it just a link in your README it will make it clean like [video](link)

Reply:
Good point. I will do it. :+1: